generate/templates/ping360.py.in

```diff
@@ -151,8 +151,27 @@ if __name__ == "__main__":
         p.transmitAngle(x)
     tend_s = time.time()
 
+    print(p)
+
     print("full scan in %dms, %dHz" % (1000*(tend_s - tstart_s), 400/(tend_s - tstart_s)))
 
-    print(p)
+    p.control_auto_transmit(0,399,1,0)
+
+    tstart_s = time.time()
+    for x in range(400):
+        p.wait_message([definitions.PING360_DEVICE_DATA])
```

Comment:
What you want to accomplish here ? Why is there a X variable and a loop for wait message ?

Reply:
i'm waiting for 400 device_data messages to arrive.

Reply:
i'll add some comments

Reply:
done?